lib/search/package-filter.js

```diff
@@ -2,14 +2,14 @@
 
 module.exports = filter
 
-function filter (data, args, notArgs) {
+function filter (data, args, notArgs, opts) {
```

Comment:
Picking up from up above, this is a hella confusing arguments list: should it be `included`, `excluded`, and `opts`?